### PR TITLE
Update de_DE translation for 3.8.2 (Meter style)

### PR DESCRIPTION
**Pull request by hoffie**
Related: #2349 #2356

### CODE REVIEW

**Review by softins — APPROVED** [review]
Sieht gut aus!

File: src/res/translation/translation_de_DE.ts [hunk 1]
```diff
@@ -1477,7 +1477,7 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="271"/>
         <source>Select the meter style to be used for the level meters. The Bar (narrow) and LEDs (round, small) options only apply to the mixerboard. When Bar (narrow) is selected, the input meters are set to Bar (wide). When LEDs (round, small) is selected, the input meters are set to LEDs (round, big). The remaining options apply to the mixerboard and input meters.</source>
-        <translation type="unfinished"></translation>
+        <translation>Wähle den Stil aus, der für die Pegel-Messer benutzt wird. Die Stile Balken (schmal) und LEDs (rund, klein) beziehen sich nur auf den Mixerbereich. Die Auswahl des Stils Balken (schmal) sorgt dafür, dass die Eingangspegelanzeigen dennoch mit breiten Balken dargestellt werden. Wenn LEDs (rund, klein) ausgwählt sind, entsprechen die Eingangspegelanzeigen dem Stil LEDs (rund, groß). Die anderen Optionen beziehen sich sowohl auf den Mixer als auch auf die Eingangspegelanzeigen.</translation>
```
Comment (ann0see) [inline src/res/translation/translation_de_DE.ts:1480]: I‘d put the style names in „ ”

**Review by ann0see — COMMENTED** [review]
Thanks!

File: src/res/translation/translation_de_DE.ts (hunk 1 above)
Comment (hoffie) [reply src/res/translation/translation_de_DE.ts]: Done. I was a bit hesitant because the same could be said about English, I guess. ;)

Comment (ann0see) [reply src/res/translation/translation_de_DE.ts]: Yes. You’re right.

**Review by ann0see — APPROVED** [review]

**Pull request merged**